force-app/main/default/customMetadata/Integration_Config.BTC_Price.md-meta.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomMetadata xmlns="http://soap.sforce.com/2006/04/metadata" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <label>BTC Price</label>
+    <protected>false</protected>
+    <values>
+        <field>Host__c</field>
+        <value xsi:nil="true"/>
+    </values>
+    <values>
+        <field>Method__c</field>
+        <value xsi:type="xsd:string">GET</value>
+    </values>
+    <values>
+        <field>Named_Credentials__c</field>
+        <value xsi:type="xsd:string">Coindesk_API</value>
+    </values>
+    <values>
+        <field>Path__c</field>
+        <value xsi:type="xsd:string">/v1/bpi/currentprice.json</value>
+    </values>
+    <values>
+        <field>Timeout__c</field>
+        <value xsi:type="xsd:double">1000.0</value>
```

Comment:
Is there a reason for this to be a double instead of a integer, thus saving the conversion in the class?

Reply:
@vhbosio this is how a Number field is represented in its metadata for Custom Metadata.
![image](https://user-images.githubusercontent.com/5306306/232167690-adb6d66c-2e97-4447-be3b-3eb28a49b08a.png)